lib/floe/cli.rb

```diff
@@ -82,5 +76,20 @@ def parse_options!(args)
 
       return workflows_inputs, opts
     end
+
+    def create_credentials(opts)
+      if opts[:credentials_given]
+        opts[:credentials] == "-" ? $stdin.read : opts[:credentials]
+      elsif opts[:credentials_file_given]
+        File.read(opts[:credentials_file])
+      else
+        {}
```

Comment:
@kbrock this looks like a functional change (nil vs {} when credentials not passed).  Was this intentional?  Not a big deal since in Context we do `@credentials = credentials || {}` anyway just curious why the change

Reply:
I'm overly sensitive right now to us making conversions (e.g.: `making JSON.parse(x) if x.kind_of?(String)` )

I'll undo this. Though I kinda like it better this way.